bootstrap_cfn/config.py

```diff
@@ -99,7 +102,22 @@ def vpc(self):
 
     def iam(self):
         # LOAD STACK TEMPLATE
-        return json.loads(pkgutil.get_data('bootstrap_cfn', 'stacks/iam.json'))
+        iam = json.loads(pkgutil.get_data('bootstrap_cfn', 'stacks/iam.json'))
+        if 'iam' in self.data:
+            if 'deploy_account' in self.data['iam']:
+                account_id = self.data['iam']['deploy_account']
+                deploy_iam = json.loads(pkgutil.get_data('bootstrap_cfn', 'stacks/iam-deploy.json'))
+                statement = deploy_iam['CrossAccountRole']['Properties']['AssumeRolePolicyDocument']['Statement']
+                statement[0]['Principal']['AWS'] = "arn:aws:iam::{0}:root".format(account_id)
+                if not 's3' in self.data:
+                    deploy_iam.pop('S3DeployUserPolicy')
+                else:
+                    bucket_name = self.data['s3']['static-bucket-name']
+                    policy = deploy_iam['S3DeployUserPolicy']['Properties']['PolicyDocument']['Statement']
+                    statement = policy[-1]
+                    statement['Resource'] = ["arn:aws:s3:::{0}/*".format(bucket_name), "arn:aws:s3:::{0}".format(bucket_name)]
```

Comment:
Now that you mention it, surely we can Ref the name of the S3 bucket too...

Will take a look :+1: